source/_patterns/01-molecules/components/highlight-item.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
In the same way as hero banner we should have a variant with each different type of content.

Reply:
> @nlisgo can you advise me what are the different content types for the Highlights? Something like that: -blog-article -collection -digest -event -interview -labs-post -podcast-episode -press-package

The list is the same as for hero banner.